lib/cylc/config.py

```diff
@@ -651,7 +652,7 @@ def __init__(self, suite, fpath, template_vars=None,
         # require reordering task_attr in lib/cylc/graphing.py).
 
         self.leaves = self.get_task_name_list()
-        for ns, ancestors in self.runtime['first-parent ancestors'].items():
+        for _, ancestors in self.runtime['first-parent ancestors'].items():
```

Comment:
yes, I should have looked more carefully (I was just blindly changing unused variables to `_`!)